src/recovery/mod.rs

```diff
@@ -233,13 +254,73 @@ impl Recovery {
             self.hystart.start_round(pkt_num);
         }
 
+        // Pacing: Set the pacing rate if BBR is not used
+        if self.pacing_enabled &&
+            (self.cc_algo == CongestionControlAlgorithm::CUBIC ||
+                self.cc_algo == CongestionControlAlgorithm::Reno)
+        {
+            let rate = match self.smoothed_rtt {
+                Some(srtt) =>
+                    ((self.congestion_window as u128 * 1000000) /
+                        srtt.as_micros()) as usize,
+                None => 0,
+            };
+            self.set_pacing_rate(rate);
+        }
```

Comment:
If you want to keep this block here, then since `set_pacing_rate()` ignores 0 values you can probably rewrite the internals to something like 

```
if let Some(srtt) = self.smoothed_rtt {
 let rate = ((self.congestion_window as u128 * 1000000) /
                 srtt.as_micros()) as usize;
 self.set_pacing_rate(rate);
}
```

Reply:
done!